includes/class-wc-shortcodes.php

```diff
@@ -565,7 +565,7 @@ public static function product_page( $atts ) {
 			$single_product = new WP_Query( $args );
 			?>
 			<script type="text/javascript">
-				jQuery( document ).ready( function( $ ) {
+				jQuery( function( $ ) {
```

Comment:
@codelingo capture use `$( handler )` instead of `$( document ).ready( handler )`